cmd/contour/servecontext_test.go

```diff
@@ -100,3 +104,98 @@ func TestServeContextTLSParams(t *testing.T) {
 		})
 	}
 }
+
+func TestConfigFileDefaultOverrideImport(t *testing.T) {
+	tests := map[string]struct {
+		yamlIn []byte
+		want   func() *serveContext
+	}{
+		"empty configuration": {
+			yamlIn: []byte(``),
+			want:   newServeContext,
+		},
+		"defaults in yaml": {
+			yamlIn: []byte(`
```

Comment:
make this a string then convert it to a []byte in in the test function below

toptip: avoid the preceeding newline with this
```
    yamlIn: ""+
`incluster: false
...

Reply:
If I do this, then goimports insists that the first line of YAML must be indented.
I don't think the beginning newline is hurting anything, and it makes the inlined YAML much more readable.